tests/unit/ReportUtilsTest.js

```diff
@@ -64,150 +64,40 @@ describe('ReportUtils', () => {
-                    avatar: {
-                        testUri: '../../../assets/images/avatars/user/default-avatar_5.svg',
-                    },
-                    login: '+18332403627@expensify.sms',
-                    accountID: 4,
-                    pronouns: undefined,
-                },
-                {
-                    displayName: 'Lagertha Lothbrok',
-                    avatar: {
-                        testUri: '../../../assets/images/avatars/user/default-avatar_6.svg',
-                    },
-                    login: 'lagertha2@vikings.net',
-                    accountID: 5,
-                    pronouns: 'She/her',
-                },
-            ]);
-        });
-
-        test('withMultiParticipantReport', () => {
-            expect(ReportUtils.getDisplayNamesWithTooltips(participantsPersonalDetails, true)).toStrictEqual([
-                {
-                    displayName: 'Ragnar',
-                    login: 'ragnar@vikings.net',
-                    avatar: {
-                        testUri: '../../../assets/images/avatars/user/default-avatar_2.svg',
-                    },
-                    accountID: 1,
-                    pronouns: undefined,
-                },
-                {
-                    displayName: 'floki@vikings.net',
-                    avatar: {
-                        testUri: '../../../assets/images/avatars/user/default-avatar_3.svg',
-                    },
-                    login: 'floki@vikings.net',
-                    accountID: 2,
-                    pronouns: undefined,
-                },
-                {
-                    displayName: 'Lagertha',
-                    avatar: {
-                        testUri: '../../../assets/images/avatars/user/default-avatar_4.svg',
-                    },
-                    login: 'lagertha@vikings.net',
-                    accountID: 3,
-                    pronouns: 'She/her',
-                },
-                {
-                    displayName: '(833) 240-3627',
-                    avatar: {
-                        testUri: '../../../assets/images/avatars/user/default-avatar_5.svg',
-                    },
-                    login: '+18332403627@expensify.sms',
-                    accountID: 4,
-                    pronouns: undefined,
-                },
-                {
-                    displayName: 'Lagertha',
-                    avatar: {
-                        testUri: '../../../assets/images/avatars/user/default-avatar_6.svg',
-                    },
-                    login: 'lagertha2@vikings.net',
-                    accountID: 5,
-                    pronouns: 'She/her',
-                },
-            ]);
+            const participants = ReportUtils.getDisplayNamesWithTooltips(participantsPersonalDetails, false);
+            expect(participants).toHaveLength(5);
+
+            expect(participants[0].avatar).toBeInstanceOf(Function);
```

Comment:
Is the object in this case considered as Function? Why is that?

Reply:
I think @koko57 you can shed some light on it? If i recall correctly from our sessions it doesn't always come as an object, especially in this Jest environment and `Function` was valid.

Reply:
As we use this svg transformer for jest, we're loading svgs like in the app, we no longer get this object {testUri: ...}. And looking on proptypes this is what we actually expect here (string or function) 🙂